x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/transform/action/UpdateTransformAction.java

```diff
@@ -186,6 +187,15 @@ public boolean equals(Object obj) {
                 && Objects.equals(authState, other.authState)
                 && getTimeout().equals(other.getTimeout());
         }
+
+        @Override
+        public boolean match(Task task) {
```

Comment:
Do we also need this in `ScheduleNowTransformAction`?

Reply:
If we do then it might be best to do that in a separate PR so we get two release notes entries and the backports are clearer (since `ScheduleNowTransformAction` doesn't exist in 7.17).

Reply:
> Do we also need this in `ScheduleNowTransformAction`?

Yes, we do.

> If we do then it might be best to do that in a separate PR so we get two release notes entries and the backports are clearer (since ScheduleNowTransformAction doesn't exist in 7.17).

Ack